.flake8

```diff
@@ -0,0 +1,9 @@
+[flake8]
+max-line-length = 99
+select: E,W,F,C,N
+exclude:
+  venv
+  .git
+  build
+  dist
+  *.egg_info
```

Comment:
This is probably for the next PR, but check out the structure laid out in https://github.com/canonical/operator-template for a more comprehensive layout where this stuff is moved into pyproject.toml and the repo is set up for unit/integration testing with tox.

Reply:
Thanks Jon. I started a new PR reorganizing the repository structure to accommodate the structure in the template in order to have a more comprehensive layout for the tests.